pkg/splunk/enterprise/util.go

```diff
@@ -1604,7 +1604,7 @@ func setInstallStateForClusterScopedApps(appDeployContext *enterpriseApi.AppDepl
 				deployInfoList[i].PhaseInfo.Phase = enterpriseApi.PhaseInstall
 				deployInfoList[i].PhaseInfo.Status = enterpriseApi.AppPkgInstallComplete
 				scopedLog.Info("Cluster scoped app installed", "app name", deployInfoList[i].AppName, "digest", deployInfoList[i].ObjectHash)
-			} else {
+			} else if deployInfoList[i].PhaseInfo.Phase != enterpriseApi.PhaseInstall || deployInfoList[i].PhaseInfo.Status != enterpriseApi.AppPkgInstallComplete {
```

Comment:
Is there a reason for this change?

Reply:
Ya, it's not related to the changes here, but addressed as a minor issue. Without this change, we were unnecessarily logging errors for the apps, that were already installed successfully as part of earlier bundles.